packages/aws-rfdk/lib/deadline/scripts/bash/configureWorker.sh

```diff
@@ -113,6 +117,13 @@ if [ ${#WORKER_POOLS[@]} -gt 0 ]; then
   "$DEADLINE_COMMAND" -SetPoolsForSlave $(IFS=, ; echo "${WORKER_NAMES[*]}") $(IFS=, ; echo "${WORKER_POOLS[*]}")
 fi
 
+# Setting listening port for the workers on this node (we cannot have multiple workers listening on the same port)
+port_offset=0
+for worker_name in "${WORKER_NAMES[@]}"; do
+  "$DEADLINE_COMMAND" -ExecuteScriptNoGui "$WORKER_LISTENING_PORT_CONFIG_SCRIPT" -n $worker_name -p $(($WORKER_LISTENING_PORT+$port_offset))
```

Comment:
What if one or several ports from the range is occupied by something? 
 Do we lost these workers?

Reply:
If a port that a worker uses to listen on is already occupied, this will set the port but the worker won't be able to access it. The customer will need to use the property on the WorkerConfiguration to set a new range that is free. I believe modifying the user data will cause the old worker to be terminated and a new host to be spun up, but this isn't easy to avoid. If the customer just uses Deadline command to change the port, they would also need to modify security groups and possibly the firewall if the worker is on Windows. All of this would cause drift between their CloudFormation templates and their deployment.